website/docs/about/support.md

```diff
@@ -1,25 +1,27 @@
 ---
 title: Support
-order: 104
+order: 103
 ---
 
+!!! Info
 
-We are happy to provide support to HashiCorp product teams that use our design system. Here are some of the ways to get help if you get stuck.
+Many resources and links on this page are only available to HashiCorp employees.
+!!!
 
-## Office Hours
+We're happy to provide support to HashiCorp product teams that use Helios. Use these resources to get help if you get stuck.
```

Comment:
```suggestion
We’re happy to provide support to HashiCorp product teams that use Helios. Use these resources to get help.
```
Including @alex-ju 's suggestion but also removing the "if you get stuck" as you don't need to be "stuck" to seek out help from us.